downstream/assemblies/platform/assembly-using-rhsso-operator-with-automation-hub.adoc

```diff
@@ -0,0 +1,37 @@
+ifdef::context[:parent-context: {context}]
+
+[id="using-rhsso-operator-with-automation-hub"]
+
+:context: using-a-rhsso-operator
+
+= Using a Red Hat Single Sign On Operator with {HubName}
+
+The _{PlatformName}_ (AAP) _{PrivateHubNameStart}_ (PAH) uses Red Hat Single Sign-On (RH-SSO) for authentication.
+
+The _Red Hat Single Sign-On_ (RH-SSO) Operator creates and manages resources.
+Use this Operator to create custom resources to automate Red Hat Single Sign-On administration in Openshift.
+
+When installing AAP on _Virtual Machines_ (VMs) the installer can automatically install and configure RH-SSO for use with PAH.
+When installing AAP on _{OCP}_ (OCP) you must install Single Sign-On seperately.
+This chapter describes the process to configure RH-SSO and integrate it with PAH when AAP is installed on {OCPShort}.
```

Comment:
I'd like to see these 3 lines separated - perhaps in separate paragraphs, or maybe the 1st 2 lines could have bullet points.